src/DocumentFormat.OpenXml/Validation/DocumentValidator.cs

```diff
@@ -38,10 +39,11 @@ public DocumentValidator(SchemaValidator schemaValidator, SemanticValidator sema
         /// </summary>
         /// <param name="document">The document to be validated.</param>
         /// <param name="settings">The settings to be used during validation.</param>
+        /// <param name="token"></param>
```

Comment:
I think we should add comments to tell the developer that the `token` do nothing in .NET 3.5

Reply:
Thanks. This method is private in. Net 3.5 so it won't be visible on that platform as there is no cancelation token there. Or are you referring to devs in the open xml codebase?

Reply:
@twsouthwick Yes, for developers.

Reply:
Sure Good idea - I'll document that